projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -108,6 +108,47 @@ <h3 *ngIf="content.title" class="cx-card-title">
             >{{ action.name }}</a
           >
         </div>
+        <div *cxFeatureLevel="'4.2'" [ngSwitch]="action.event">
+          <button
+            *ngSwitchCase="'delete'"
+            class="cx-card-link card-link btn-link cx-action-link"
+            (click)="delete()"
+            (keydown.enter)="delete($event)"
+          >
+            {{ action.name }}
+          </button>
+          <button
+            *ngSwitchCase="'default'"
+            class="cx-card-link card-link btn-link cx-action-link"
+            (click)="setDefault()"
+            (keydown.enter)="setDefault()"
+          >
+            {{ action.name }}
+          </button>
+          <button
+            *ngSwitchCase="'send'"
+            class="cx-card-link card-link btn-link cx-action-link"
+            (click)="send()"
+            (keydown.enter)="send()"
+          >
+            {{ action.name }}
+          </button>
+          <button
+            *ngSwitchCase="'edit'"
+            class="cx-card-link card-link btn-link cx-action-link"
+            (click)="edit()"
+            (keydown.enter)="edit($event)"
+          >
+            {{ action.name }}
+          </button>
+          <a
+            *ngSwitchDefault
+            href="{{ action.link }}"
+            class="card-link btn-link cx-action-link"
```

Comment:
@developpeurweb Can we reduce the classes used here? Are they bootstrap?

Reply:
Yes, the first 2 are bootstrap. 
If this is going to remain as a "link" for the next minor, then you can safely remove the `card-link` class **only**, but the rest have to stay to avoid patching the styles.
All these will have to be `<button>`s for the next major.

Reply:
Please ignore my previous comment. On second thought, this would be a breaking change for any client using bootstrap-based styles, so let's leave it as is for now and change it for good in 5.0.